tests/dcps_tests.lst

```diff
@@ -237,9 +237,12 @@ tests/DCPS/EntityLifecycleStress/run_test.pl rtps_disc publishers 10 subscribers
 tests/DCPS/LivelinessKeepAliveTest/run_test.pl: !DCPS_MIN !OPENDDS_SAFETY_PROFILE
 tests/DCPS/LivelinessTimeout/run_test.pl: !DCPS_MIN !GH_ACTIONS
 tests/DCPS/LivelinessTimeout/run_test.pl rtps_disc: !DCPS_MIN RTPS
-tests/DCPS/UnitTests/run_test.pl: !DCPS_MIN !GH_ACTIONS
-tests/unit-tests/dds/DCPS/run_test_unit.pl: !DCPS_MIN !GH_ACTIONS
-tests/unit-tests/dds/DCPS/transport/framework/run_test_unit.pl
+tests/DCPS/BitDataReader/run_test.pl: !DCPS_MIN !GH_ACTIONS
```

Comment:
Does it need to have `!GH_ACTIONS`?

Reply:
I went with what was there.  I can remove it and see what happens.